e2e/cypress.json

```diff
@@ -10,6 +10,8 @@
     "env": {
         "storybookUrl": "http://localhost:6006/",
         "webhookBaseUrl": "http://localhost:3000",
-        "setChromeWebSecurity": true
```

Comment:
can we do that, @saturninoabril ? For SAML I had a cypress.okta.json file with setChromeWebSecurity: false to override this.

Reply:
Nope. I was running SAML before I checked in and forgot to revert this. Nice catch.